packages/common/src/ContractInteractor.ts

```diff
@@ -230,6 +229,12 @@ export class ContractInteractor {
     if (block.baseFeePerGas != null) {
       this.logger.info('Network supports Type 2 Transactions (EIP-1559). Checking RPC node \'eth_feeHistory\' method')
       try {
+        const { baseFeePerGas, priorityFeePerGas } = await this.getGasFees(constants.getGasFeesBlocks, constants.getGasFeesPercentile)
```

Comment:
if you're using here the constants, then what's the use of having them configured at all?